src/nodes.coffee

```diff
@@ -4483,6 +4478,18 @@ exports.StringWithInterpolations = class StringWithInterpolations extends Base
         delete node.comments
       return yes
 
+    elements
+
+  compileNode: (o) ->
+    @comments ?= @startQuote?.comments
```

Comment:
After fixing the one test by using exclusive location data when mapping comments (see previous comment in `helpers.coffee`), a couple other comment-related tests were breaking

They were able to pass by preserving comments that got attached to the "start quote" (`STRING_START`) of interpolated strings

So that's why `startQuote` gets passed in from the grammar now when constructing `StringWithInterpolations`, so that its comments can be preserved and output here